webpack.config.js

```diff
@@ -160,22 +163,45 @@ module.exports = ( env ) => {
     }
   }
 
+  function getCustomIfUtils( { ifDev, ifProd, ifSite } = {} ) {
 
-  function ifProdOrSite( returnValue = true ) {
+    return {
+
```

Comment:
for the next time, we don't need to produce 1000 lines of blank lines, mario 200px font era is long gone. 

Also it's a smell if you need lot of blank lines in the function to have separated content, it probably does too much and should be chunked to separate small footprint functions